src/frequenz/sdk/microgrid/_data_pipeline.py

```diff
@@ -21,9 +21,9 @@
 from ..actor._actor import Actor
 from ..microgrid.component import Component
 from ..timeseries._grid_frequency import GridFrequency
+from ..timeseries.grid import Grid
+from ..timeseries.grid import get as get_grid
```

Comment:
This can probably be done on the same line:
```suggestion
from ..timeseries.grid import get as get_grid, Grid
```

Reply:
`isort` will complain then. So I reverted that again.